caPutLogApp/caPutLogTask.c

```diff
@@ -248,8 +261,10 @@ static void caPutLogTask(void *arg)
 
     while (caPutLogConfig != caPutLogNone) {                 /* Main Server Loop */
 
+        timeout = burstTimeout;
+
         /* Receive next message */
-        msg_size = epicsMessageQueueReceiveWithTimeout(caPutLogQ, &pnext, MSG_SIZE, 5.0);
+        msg_size = epicsMessageQueueReceiveWithTimeout(caPutLogQ, &pnext, MSG_SIZE, timeout);
```

Comment:
Why introduce the `double timeout` variable here? I don't see any reason to not just use `burstTimeout` directly in the call to `epicsMessageQueueReceiveWithTimeout()`.